src/shogun/clustering/KMeansMiniBatch.cpp

```diff
@@ -53,41 +53,39 @@ void KMeansMiniBatch::minibatch_KMeans()
 		              "iterations {} ",
 		max_iter);
 
-	auto lhs=
-		distance->get_lhs()->as<DenseFeatures<float64_t>>();
+	auto lhs = distance->get_lhs()->as<DenseFeatures<float64_t>>();
+
 	auto rhs_mus = std::make_shared<DenseFeatures<float64_t>>(cluster_centers);
-	auto rhs_cache=distance->replace_rhs(rhs_mus);
+    distance->replace_rhs(rhs_mus);
+
 	int32_t XSize=lhs->get_num_vectors();
-	int32_t dims=lhs->get_num_features();
 
 	SGVector<float64_t> v=SGVector<float64_t>(k);
 	v.zero();
 
 	for (auto i : SG_PROGRESS(range(max_iter)))
 	{
-		SGVector<int32_t> M=mbchoose_rand(batch_size,XSize);
-		SGVector<int32_t> ncent=SGVector<int32_t>(batch_size);
-		for (int32_t j=0; j<batch_size; j++)
+		SGVector<int32_t> M = mbchoose_rand(batch_size, XSize);
```

Comment:
It makes reviewing new code really hard if you have a lot of added whitespace changes ....

Reply:
(because it is hard to see what you added)

Reply:
I will make a commit with unnessesary changes.

Reply:
OK

Reply:
I reverted all unnecessary changes.

Reply:
thanks. Feel free to send the commit later once we merged this here. but not important